wwr_ui/randomizer_window.py

```diff
@@ -109,8 +109,8 @@ def __init__(self, cmd_line_args=OrderedDict()):
     # Hide unfinished options from the GUI (still accessible via settings.txt).
     self.ui.randomize_bgm.hide()
     
-    self.show()
-    
+    self.show()    
+
```

Comment:
You probably don't want to make changes to these two lines.

Reply:
That one was actually just an accident when testing stuff, but it didn't seem to break anything.